Jenkinsfile

```diff
@@ -168,6 +169,18 @@ pipeline {
 
                     sudo sh -c "curl -s -L https://github.com/mikefarah/yq/releases/download/3.3.2/yq_linux_amd64 > /usr/local/bin/yq"
```

Comment:
let's switch to yq 4

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/21/commits/ce9dbd37615ce61ca476bb38d31829eadc785ddd